### PR TITLE
 Repo links

### PR DESCRIPTION
**Pull request by jburel**
Point to release versions
cc @mtbc

### CODE REVIEW

File: omero/developers/Model/Units.rst [hunk 1]
```diff
@@ -13,13 +13,13 @@ converting their internal unit to the OME default.
...
 ---------------
 
-* :javadoc:`Electric potential <omero/model/ElectricPotentialI.html>`
-* :javadoc:`Frequency <omero/model/FrequencyI.html>`
-* :javadoc:`Length <omero/model/LengthI.html>`
-* :javadoc:`Power <omero/model/PowerI.html>`
-* :javadoc:`Pressure <omero/model/PressureI.html>`
-* :javadoc:`Temperature <omero/model/TemperatureI.html>`
-* :javadoc:`Time <omero/model/TimeI.html>`
+* :javadoc_model:`Electric potential <omero/model/ElectricPotentialI.html>`
```
Comment (mtbc) [inline omero/developers/Model/Units.rst]: Probably needs to be Blitz. To get this PR in might should first have a CI job that actually checks links.

File: omero/conf.py [hunk 2]
```diff
@@ -97,39 +97,48 @@
...
+    'server_source' : (omero_subs_github_root.format('omero-server', 'blob', 'v'+conf_autogen.version_server), ''),
+    'server_sourcedir' : (omero_subs_github_root.format('omero-server', 'tree', 'v'+conf_autogen.version_server), ''),
+    'blitz_source' : (omero_subs_github_root.format('omero-blitz', 'blob', 'v'+conf_autogen.version_blitz), ''),
+    'blitz_sourcedir' : (omero_subs_github_root.format('omero-blitz', 'tree', 'v'+conf_autogen.version_blitz), ''),
+    'java_gateway_source' : (omero_subs_github_root.format('omero-java-gateway', 'blob', 'v'+conf_autogen.version_gateway), ''),
+    'java_gateway_sourcedir' : (omero_subs_github_root.format('omero-java-gateway', 'tree', 'v'+conf_autogen.version_gateway), ''),
+    'matlab_source' : (omero_subs_github_root.format('omero-matlab', 'blob', 'v'+conf_autogen.version_matlab), ''),
+    'matlab_sourcedir' : (omero_subs_github_root.format('omero-matlab', 'tree', 'v'+conf_autogen.version_matlab), ''),
+    'insight_source' : (omero_subs_github_root.format('omero-insight', 'blob', 'v'+conf_autogen.version_matlab), ''),
```
Comment (mtbc) [inline omero/conf.py]: Should be "insight".

File: omero/developers/Model.rst [hunk 3]
```diff
@@ -78,32 +78,35 @@ exceptions in the model.
...
 source directory are the starting point for all code generation. Currently
-however, the files under :model_sourcedir:`src/main/resources/mappings`
+however, the files in :omero_subs_github_repo_root:`omero-model` under :model_sourcedir:`src/main/resources/mappings`
 are hand-written based on the XSD files.
 
-The ant task created from the :dsl_plugin_sourcedir:`src` Java files
-is then used to turn the mapping files into generated Java code under the
-:file:`model/target/generated/src` directory. These classes are all within the
+The task created from the :dsl_plugin_sourcedir:`src` Java files
+is then used to turn the mapping files into generated Java code in in :omero_subs_github_repo_root:`omero-model` under the
```
Comment (mtbc) [inline omero/developers/Model.rst]: too many "in"

File: omero/developers/Model.rst [hunk 4]
```diff
@@ -78,32 +78,35 @@ exceptions in the model.
...
 The above classes are considered the internal server code, and are the only
 objects which can take part in Hibernate transactions.
 
-External to the server code is the "blitz" layer. These classes are in the
-omero.model package. They are generated by another call to the DSL ant task
+External to the server code is the :omero_subs_github_repo_root:`omero-blitz` layer. These classes are in the
+omero.model package. They are generated by another call to the DSL task
 in order to generate the Java, Python, C++, and Ice files under
-:file:`src/generated`.
+:file:`build/src/`.
```
Comment (mtbc) [inline omero/developers/Model.rst]: maybe `build/psql/`?

File: omero/developers/Server/LightAdmins.rst [hunk 5]
```diff
@@ -206,7 +206,7 @@ indirect means of authenticating legitimately set values. Each running
 server has a secret key recorded in the `uuid` property of :ref:`Node
 <OMERO model class Node>`. This key is not available to OMERO clients,
 it is internal to the server. To set a new file's ``repo`` the
-:javadoc:`repository DAO
+:javadoc_server:`repository DAO
```
Comment (mtbc) [inline omero/developers/Server/LightAdmins.rst]: Should be blitz.

File: omero/developers/Server/Properties.rst [hunk 6]
```diff
@@ -6,7 +6,12 @@ distributions, several files are provided which help to configure
 OMERO.server:
 
 :source:`etc/omero.properties`
-  Our central configuration file with all defaults
+  Since 5.5, the following repositories have now 
+  a ``properties`` file with the properties used in the repository itself.
+  See :model_source:`omero-model.properties <src/main/resources/omero-model.properties>`,
+  :common_source:`omero-common.properties <src/main/resources/omero-common.properties>`,
+  :server_source:`omero-server.properties <src/main/resources/omero-server.properties>`,
+  :blitz_source:`omero-model.properties <src/main/resources/omero-blitz.properties>`.
```
Comment (mtbc) [inline omero/developers/Server/Properties.rst]: blitz, not model

File: omero/developers/build-system.rst [hunk 7]
```diff
@@ -7,8 +7,12 @@ Build System
 
 .. _Ant: https://ant.apache.org
 .. _Ivy: https://ant.apache.org/ivy
+.. _Gradle: https://gradle.org/
 
-OMERO mostly uses an Ant_-based build with dependency management provided by
+Since 5.5, OMERO decouples many components and uses, for some components, an Gradle_-based build. The two overarching repositories are
+:omero_subs_github_repo_root:`omero-build` and :omero_subs_github_repo_root:`omero-gradle-plugins`. 
+See the README of each repositories for more details.
```
Comment (mtbc) [inline omero/developers/build-system.rst]: repository

File: omero/sysadmins/cli/config.rst [hunk 8]
```diff
@@ -34,9 +34,11 @@ different profile, e.g.::
 The values set via :program:`omero config set` override those compiled into the
 server jars. The default values which are set can be seen in
 :doc:`/sysadmins/config`. To add several values to a configuration, you can
-pipe them via standard in using :program:`omero config load`::
+pipe them via standard in using :program:`omero config load`. To grep for the example
+the LDAP configuration from
```
Comment (mtbc) [inline omero/sysadmins/cli/config.rst]: maybe remove "the"?

File: omero/conf.py [hunk 9]
```diff
@@ -97,39 +97,48 @@
...
     'javadoc' : (downloads_root + '/latest/omero5.4/api/%s', ''),
+    'javadoc_gateway_java' : (docs_root + '/omero-gateway/' + conf_autogen.version_gateway + '/javadoc/%s', ''),
+    'javadoc_blitz' : (docs_root + '/omero-blitz/' + conf_autogen.version_blitz + '/javadoc/%s', ''),
+    'javadoc_server' : (docs_root + '/omero-server/' + conf_autogen.version_server + '/javadoc/%s', ''),
+    'javadoc_renderer' : (docs_root + '/omero-renderer/' + conf_autogen.version_renderer + '/javadoc/%s', ''),
+    'javadoc_romio' : (docs_root + '/omero-romio/' + conf_autogen.version_romio + '/javadoc/%s', ''),
+    'javadoc_common' : (docs_root + '/omero-common/' + conf_autogen.version_common + '/javadoc/%s', ''),
+    'javadoc_model' : (docs_root + '/omero-model/' + conf_autogen.version_model + '/javadoc/%s', ''),
+    'slicedoc_blitz' : (docs_root + '/omero-blitz/' + conf_autogen.version_blitz + '/slice2html/%s', ''),
+    'slicedoc_server' : (docs_root + '/omero-server/' + conf_autogen.version_server + '/slice2html/%s', ''),
```
Comment (mtbc) [inline omero/conf.py]: Is there any?

Comment (jburel) [reply omero/conf.py]: no I will remove it. This was prep before I went into the deep end of the doc

**Comment by jburel** [comment]
@mtbc ``linkcheck`` is now run
and it is green see https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-docs/35/

File: omero/developers/Server/Properties.rst [hunk 10]
```diff
@@ -6,7 +6,12 @@ distributions, several files are provided which help to configure
 OMERO.server:
 
 :source:`etc/omero.properties`
-  Our central configuration file with all defaults
+  Since 5.5, the following repositories have now
```
Comment (mtbc) [inline omero/developers/Server/Properties.rst]: now have

File: omero/developers/build-system.rst [hunk 11]
```diff
@@ -164,7 +152,6 @@ The OMERO build system defines and uses four types of Ivy
 :ivydoc:`configurations <ivyfile/configurations.html>`:
 
 #. build: defines dependencies to be used for building
-#. server: defines dependencies to be bundled under :file:`lib/server`
```
Comment (mtbc) [inline omero/developers/build-system.rst]: Interesting as we still do get dependencies under there.

File: omero/developers/build-system.rst [hunk 12]
```diff
@@ -283,10 +264,8 @@ Building OMERO
...
-#. adjust the ``versions.bioformats`` property under
-   :file:`etc/omero.properties` to the version chosen for the Bio-Formats
+#. adjust the version of ``ome:formats-gpl`` in
+   :model_source:`build.gradle` to the version chosen for the Bio-Formats
    build
 
-#. run the build system as usual::
-
-     $ ./build.py build-dev
+#. publish locally :omero_subs_github_repo_root:`omero-model`
```
Comment (mtbc) [inline omero/developers/build-system.rst]: Seems an odd place to end.

File: omero/developers/logging.rst [hunk 13]
```diff
@@ -19,7 +19,7 @@ size are limited.
 :source:`logback-cli.xml<etc/logback-cli.xml>`
 controls the output for the command line importer: all logging goes
 to standard err, while useful output (pixel ids, or used files) goes to
-standard out. It is contained within the blitz.jar itself. Therefore, to
+standard out. It is contained within the ``omero-blitz.jar`` itself. Therefore, to
```
Comment (mtbc) [inline omero/developers/logging.rst:22]: I can't find it there. Not this PR's fault though.

**Comment by mtbc** [comment]
Seems good to me. Will mention in Slack in case anyone else wants a quick last look!

**Comment by jburel** [comment]
Thanks @mtbc for reviewing

**Pull request merged**